test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -309,6 +312,26 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 		})
 	}
 
+	// TODO consider moving to [Feature:ClusterSizeAutoscalingGpu] as soon as NAP goes out of beta. Currently
+	// project needed to run the NAP tests require whitelisting for NAP alpha
+	It("NAP should add a pool with GPUs if unschedulable POD which require GPU exists [Feature:ClusterSizeAutoscalingScaleWithNAP]", func() {
+		framework.SkipUnlessProviderIs("gke")
+
+		installNvidiaDriversDaemonSet()
+		framework.ExpectNoError(enableAutoprovisioning(`"resource_limits":{"name":"nvidia-tesla-k80", "minimum":0, "maximum": 3}`))
```

Comment:
This won't work without limits for CPU and memory, unfortunately:( (see enableAutoprovisioning implementation...)